app/controllers/submit.js

```diff
@@ -237,7 +237,15 @@ export default Controller.extend(Analytics, BasicsValidations, COIValidations, N
     // Must have year and copyrightHolders filled if those are required by the licenseType selected
     licenseValid: false,
 
-    _names: ['Server', 'File', 'Assertions', 'Basics', 'Discipline', 'Authors', 'COI', 'Supplemental'], // Form section headers
+    _names: computed('shouldShowCoiPanel', 'shouldShowAuthorAssertionsPanel', function() {
+        if (this.get('shouldShowCoiPanel')) {
```

Comment:
Might need to switch the order of the if statements here. In the case that both `shouldShowCoiPanel` and `shouldShowAuthorAssertionsPanel` are true, it will still only stop at the first case and not hit the necessary one.
```suggestion
        if (this.get('shouldShowCoiPanel') && this.get('shouldShowAuthorAssertionsPanel')) {
            return ['Server', 'File', 'Assertions', 'Basics', 'Discipline', 'Authors', 'COI', 'Supplemental'];
        }
        if (this.get('shouldShowCoiPanel')) {
            return ['Server', 'File', 'Basics', 'Discipline', 'Authors', 'COI', 'Supplemental'];
        }
```